src/kibana/partials/saved_object_finder.html

```diff
@@ -2,6 +2,7 @@
   <div class="form-group finder-form">
     <div>
       <a class="small pull-right" ng-click="manageObject(type)">manage {{type}}</a>
+      <div class="clearfix"></div>
```

Comment:
clearfix makes me :cry:

Can we name this container and make it a flexbox too?